docs/Running-Mastodon/Production-guide.md

```diff
@@ -49,7 +57,7 @@ server {
     proxy_set_header X-Real-IP $remote_addr;
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     proxy_set_header X-Forwarded-Proto https;
-
+    proxy_set_header Proxy "";
```

Comment:
It's to mitigate a Nginx vulnerability  : https://httpoxy.org/
Rail seem not impacted, but I think, it's a good practice.